tmt/schemas/plan.yaml

```diff
@@ -27,6 +27,10 @@ properties:
   description:
     $ref: "/schemas/core#/definitions/description"
 
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#contact
+  contact:
+    $ref: "/schemas/common#/definitions/one_or_more_strings"
```

Comment:
I guess we should use the definition from the `core` here, right? Similar as above for the `description`.

Reply:
Yes, updated.